stable/apm-server/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v1
 description: The server receives data from the Elastic APM agents and stores the data into a datastore like Elasticsearch
 icon: https://www.elastic.co/assets/blt47799dcdcf08438d/logo-elastic-beats-lt.svg
 name: apm-server
-version: 0.1.0
-appVersion: 6.2.4
+version: 0.1.1
```

lets bump to 1.0.0 since this chart is in the stable folder
```suggestion
version: 1.0.0
```

---

bumped to 1.0.0 :)